motioneye/templates/main.html

```diff
@@ -136,11 +136,11 @@
                             <td class="settings-item-value"><input type="text" class="range styled prefs" id="resolutionDimmerSlider"></td>
                             <td><span class="help-mark" title="dims the actual resolution of all cameras to save network bandwidth and traffic (doesn't work on simple MJPEG cameras)">?</span></td>
                         </tr>
+                        <tr class="settings-item">
+                            <td colspan="100"><div class="settings-item-separator" style="margin: 0px 0px 1.5em 0px;"></div></td>
+                        </tr>
                     </table>
 
-                    <tr class="settings-item">
-                        <td colspan="100"><div class="settings-item-separator" style="margin: 0px 0px 1.5em 0px;"></div></td>
-                    </tr>
 
```

Comment:
```suggestion
```
Removing doubled empty line.

Reply:
FYI: If we were to turn on https://pre-commit.ci/ then these corrections would be made automatically.

Reply:
FYI: If we were to turn on https://pre-commit.ci/ then these corrections could be made automatically.

Reply:
Can be also done with other tools via GitHub actions, I guess also with a dedicated pre-commit workflow which triggers a PR as GitHub bot when a diff was found. Generally for now I think we should stay with GitHub actions and not switch to/include further external CI/CD providers, just to keep it transparent and simple to connect workflows with each other at some point.